Gruntfile.js

```diff
@@ -1,200 +1,8 @@
-var packageJson = require('./package.json');
-
 /* globals module */
 module.exports = function(grunt) {
 
-  var COPYRIGHT_TEXT = grunt.file.read('lib/copyright.frag');
```

Comment:
This is great! It's nice to see we're able to remove so much of this boilerplate code.